File: 02 Januar 2025/WebTemplate/WebTemplate/Models/Kola.cs
namespace WebTemplate.Models;

public class Kola
{
    [Key]
    public int ID { get; set; }

    public required uint Kilometraza { get; set; }

    [Range(1930, 2026)]
    public required uint Godiste { get; set; }

    public required uint BrojSedista { get; set; }
    public required uint CenaPoDanu { get; set; }
    
    public  Modeli? Model { get; set; }

    [ForeignKey("IznajmljivanjeFK")]
    public Iznajmljivanja? Iznajmljivanje { get; set; }

    public required bool DaLiJeIznajmljen{ get; set; }

}

File: 02 Januar 2025/Front/App/Kola.js

export class Kola{
    
    constructor(Kola){
        this.id = Kola.id;
        this.model = Kola.model;
        this.kilometraza = Kola.kilometraza;
        this.godiste = Kola.godiste;
        this.brojSedista = Kola.brojSedista;
        this.cenaPoDanu = Kola.cenaPoDanu;
        this.iznajmljen = Kola.daLiJeIznajmljen;
        
    }

    draw(container)
    {
        const kolaDiv = document.createElement("div");
        kolaDiv.classList("kola-div")
        container.appendChild(kolaDiv);
        
    }
}

